Chapter_2/Readme.md

```diff
@@ -0,0 +1,3 @@
+#Can you find a key combo in RubyMine to comment out a line of code?
+    By using the # character you can comment out a specific line in Rubymine
```

Comment:
There's a keyboard shortcut.  Look in your Keymap for `Comment with Line Comment`.